src/track/beatmap.cpp

```diff
@@ -631,7 +631,7 @@ void BeatMap::scaleQuadruple() {
         // Need to not accrue fractional frames.
         int distance = it->frame_position() - prevBeat.frame_position();
         Beat beat;
-        for (i=1; i<=3, i++;) {
+        for (int i=1; i<=3, i++;) {
```

Comment:
Previous conversions appear in the manual (p43). Necessary to update this after merge?